packages/itwinui-css/src/tabs/tabs.scss

```diff
@@ -16,6 +15,8 @@
     display: flex;
     align-items: center;
     font-size: $iui-font-size;
+    min-height: $iui-component-height;
+    height: 100%;
     cursor: pointer;
     @media (prefers-reduced-motion: no-preference) {
```

Comment:
We can remove the `@media (prefers-reduced-motion: no-preference)` here and in other places that aren't effecting motion.